cmd/cluster.go

```diff
@@ -689,6 +692,10 @@ WARNING: You are going to create a cluster that has no default internet access w
 		}
 	}
 
+	if viper.IsSet("pod-PID-limit") {
```

Comment:
We have only lower-case on CLI flags.

```suggestion
	if viper.IsSet("pod-pid-limit") {
```

We should also only apply this in combination with `--yes-i-really-mean-it` as this is potentially a dangerous setting that prevents pods from starting in a cluster.